net-dns/maradns/maradns-3.5.0020.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+inherit python-any-r1 systemd toolchain-funcs
+
+DESCRIPTION="A security-aware DNS server"
+HOMEPAGE="https://maradns.samiam.org"
+SRC_URI="https://maradns.samiam.org/download/${PV%.*}/${PV}/${P}.tar.xz"
+
+# The GPL-2 covers the init script, bug 426018.
+LICENSE="BSD-2 GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~x86"
+IUSE="examples ipv6"
+
+BDEPEND="${PYTHON_DEPS}
+	dev-lang/perl"
+DEPEND="
+	acct-group/maradns
+	acct-user/duende
+	acct-user/maradns"
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-flags-ipv6.patch
+)
+
+src_prepare() {
+	default
+	python_fix_shebang tools/bind2csv2.py
+}
+
+src_configure() {
+	tc-export CC
+	./configure \
+		$(usex ipv6 --ipv6 "") || die "Failed to configure"
+}
+
+src_install() {
+	# Install the MaraDNS and Deadwood binaries
+	dosbin server/maradns
+	dosbin tcp/zoneserver
+	dosbin deadwood-${PV}/src/Deadwood
+	dobin tcp/{getzone,fetchzone}
+	dobin tools/{askmara,askmara-tcp,duende,bind2csv2.py,csv1tocsv2.pl}
+
+	# MaraDNS docs, manpages, misc
+	docompress -x /usr/share/doc/${PF}/maradns.gpg.key
+	dodoc {CHANGELOG,COPYING,SUPPORT,maradns.gpg.key}
+	dodoc doc/en/{QuickStart,faq.*,*.md,README}
+	dodoc -r doc/en/{text,webpage,tutorial}
+	docinto deadwood
+	dodoc deadwood-${PV}/doc/{*.txt,*.html,CHANGELOG,Deadwood-HOWTO}
+	dodoc -r deadwood-${PV}/doc/internals
+
+	# Install examples (optional)
+	if use examples ; then
+		docinto examples
+		dodoc doc/en/examples/example_*
+	fi
+
+	# Install manpages
+	doman doc/en/man/*.[1-9]
+	doman deadwood-${PV}/doc/{Deadwood,Duende}.1
+
+	# Example configurations.
+	insinto /etc/maradns
+	newins doc/en/examples/example_full_mararc mararc_full.dist
```

Comment:
Should you advertise about mararc.sample in `pkg_postinst`?

Reply:
OK, done